release_notes.md

```diff
@@ -3,6 +3,8 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+- Update Java Worker Version to [2.17.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.17.0)
```

Comment:
Same comment as the other PR. This should replace the previous entry. Can you please remove the other one?